analytics/views.py

```diff
@@ -1157,6 +1158,14 @@ def support(request: HttpRequest) -> HttpResponse:
             elif status == "deactivated":
                 do_deactivate_realm(realm, request.user)
                 context["message"] = f"{realm.name} deactivated."
+        elif request.POST.get("billing_method", None) is not None:
+            billing_method = request.POST.get("billing_method")
+            if billing_method == "send_invoice":
+                update_billing_method_of_current_plan(realm, False)
+                context["message"] = f"Billing method of {realm.name} updated to pay by invoice."
+            elif billing_method == "charge_automatically":
+                update_billing_method_of_current_plan(realm, True)
+                context["message"] = f"Billing method of {realm.name} updated to charge automatically."
```

Comment:
```suggestion
        elif request.POST.get("billing_method", None) is not None:
            billing_method = request.POST.get("billing_method")
            if billing_method == "send_invoice":
                update_billing_method_of_current_plan(realm, charge_automatically=False)
                context["message"] = f"Billing method of {realm.name} updated to pay by invoice."
            elif billing_method == "charge_automatically":
                update_billing_method_of_current_plan(realm, charge_automatically=True)
                context["message"] = f"Billing method of {realm.name} updated to charge automatically."
```

will be clearer without needing to check the function signature for what the boolean  is.

Reply:
Taken care of this.